src/components/widget-info/component.js

```diff
@@ -0,0 +1,64 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import DownloadLink from 'components/link';
+import Button from 'components/button';
+import styles from './style.module.scss';
+import Info from './info.svg';
+import ArrowRight from './arrow-right.svg';
+
+class WidgetInfo extends PureComponent {
+  static propTypes = {
+    openInfoPanel: PropTypes.func,
+    data: PropTypes.arrayOf(PropTypes.shape({})),
+    filename: PropTypes.string
+  }
+
+  static defaultProps = {
+    data: null,
+    filename: null,
+    openInfoPanel: () => null
+  }
+
+  clickHandler = () => {
+    const { openInfoPanel } = this.props;
+    openInfoPanel();
+  }
+
+  render() {
+    const { data, filename } = this.props;
+    return (
+      <div className={styles.info}>
+        {filename && (
```

Comment:
In my opinion is better practice to check within the component for the property it needs. DownloadLink checks for filename if it is not defined it returns null.